packages/app/src/pages/landelijk/gedrag.tsx

```diff
@@ -239,7 +235,7 @@ export default function BehaviorPage(
             {...timelineProp}
             currentId={currentId}
             setCurrentId={setCurrentId}
-            useDatesAsRange={behaviorAnnotationsFeature.isEnabled}
+            useDatesAsRange={true}
```

Comment:
Not really a suggestion, more a 'you can do this differently':

```suggestion
            useDatesAsRange
```

In React if the prop is `true`, you can just provide the prop without value. However that might feel like it breaks consistency and can add to confusion, so I totally get it if you want to just ignore this comment. 😁